rest-service/manager_rest/rest/rest_decorators.py

```diff
@@ -123,6 +125,7 @@ class must have a "resource_fields" class variable
 
     def __call__(self, f):
         @wraps(f)
+        @traces('marshal_with {}'.format(self.response_class.__name__))
```

Comment:
Every time the interpreter decorates the function. I think only once since it's imported at the endpoint mapper (via import of the rest handlers classes)

Reply:
Is that OK then?